From 9dea9338b8bc3b4e560575b3c161ea7560dfa438 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 18 Jun 2025 21:21:31 +0200
Subject: [PATCH] Revert truststore extra to enable noarch

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index c9833dd..52440ee 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,7 +32,6 @@ dependencies = [
     "python-dotenv>=0.15",
     "resolvelib>=1.1",
     "installer<0.8,>=0.7",
-    "truststore>=0.10.4; python_version >= \"3.10\"",
     "tomli>=1.1.0; python_version < \"3.11\"",
     "importlib-metadata>=3.6; python_version < \"3.10\"",
     "hishel[httpx]>=1.0.0",
@@ -86,8 +85,9 @@ msgpack = [
 template = [
     "pdm[copier,cookiecutter]",
 ]
+truststore = ["truststore; python_version >= \"3.10\""]
 all = [
-    "pdm[keyring,template]",
+    "pdm[keyring,template,truststore]",
 ]
 locked = [
     "anyio==4.9.0",
